packages/backend/apps/multitenancy/services/membership.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.auth import get_user_model
+
+from ..models import Tenant, TenantMembership
+from ..constants import TenantUserRole
+
+User = get_user_model()
+
+
+def create_tenant_membership(user: User, tenant: Tenant, role: TenantUserRole = TenantUserRole.OWNER):
+    membership = TenantMembership.objects.create(user=user, tenant=tenant, role=role)
```

Comment:
Yes, I know about that. Actually I did it for a reason. I created it as a service because I'm expecting that many different actions could be ran here e.g. sending emails/notifications about membership. Maybe some logic of recalculating tenant plan. That is why I didn't make this service as a oneliner - for future possible purposes.